src/index.js

```diff
@@ -526,7 +605,7 @@ const handler = (event, context) => {
   }
 
   if (event.Records) {
-    handleRecords(event, context, lintEnv);
+    handleRecords(event, context, lintEnv, processS3BucketLogs);
```

Comment:
I think we no need to pass this,
process.env this can be accessed anywhere across the file. Lets make handleRecords generic